eclair-core/src/main/scala/fr/acinq/eclair/channel/Helpers.scala

```diff
@@ -833,6 +833,58 @@ object Helpers {
       irrevocablySpent.contains(outPoint)
     }
 
+    /**
+      * This helper function returns the fee paid
+      *
+      * @return
+      */
+    def networkFeePaid(tx: Transaction, d: DATA_CLOSING): Option[(Satoshi, String)] = {
```

Comment:
nitpick: scaladoc comment is quite minimalistic (though what the code does is pretty clear)